### PR TITLE
Enhanced docs with fields and issues related to invalid resume token

### PR DESCRIPTION
**Pull request by tomasz-sadura**
…for MongoDB source connector

### CODE REVIEW

**Review by bochenekmartin — APPROVED** [review]

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc [hunk 1]
```diff
@@ -121,6 +121,18 @@ on the *Create Connector* page:
 | `Error tolerance`
 | `mongo.errors.tolerance`
 | Error tolerance response during connector operation. Default value is `none` and signals that any error will result in an immediate connector task failure. Value of `all` changes the behavior to skip over problematic records.
+
+| `Heartbeat interval milliseconds`
+| `heartbeat.interval.ms`
+| The length of time between sending heartbeat messages to record the post batch resume token when no source records have been published. Improves the resumability of the connector for low volume namespaces. Use 0 to disable.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc]:
```suggestion
| The length of time it takes when sending heartbeat messages to record the post-batch resume token when no source records have been published. Improves the resumability of the connector for low volume namespaces. Specify `0` to disable.
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc [hunk 2]
```diff
@@ -121,6 +121,18 @@ on the *Create Connector* page:
...
+
+| `The heartbeat topic name`
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc]:
```suggestion
| `heartbeat topic name`
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc [hunk 3]
```diff
@@ -121,6 +121,18 @@ on the *Create Connector* page:
...
+| `heartbeat.topic.name`
+| The name of the topic to publish heartbeats to. Defaults to `__mongodb_heartbeats`.
+
+| `Offset partition name`
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc:133]:
```suggestion
| `offset partition name`
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc [hunk 4]
```diff
@@ -121,6 +121,18 @@ on the *Create Connector* page:
...
+| `offset.partition.name`
+| Use a custom offset partition name. If blank the default partition name based on the connection details will be used.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc]:
```suggestion
| Use to specify a custom offset partition name. If blank, the default partition name based on the connection details is used.
```

**Comment by Feediver1** [comment]
Hi @tomasz-sadura ,
Not sure why this is a draft? Are you awaiting additional updates? I can commit the edits and merge if you are ready to add this content. Please advise. Thx!

**Comment by tomasz-sadura** [comment]
I'm ok with committing the edits, but the changes are not yet deployed to production.

**Comment by Feediver1** [comment]
Please ping me for approval when ready for merge/the updates are deployed in production. Thx

**Review by Feediver1 — APPROVED** [review]

**Pull request merged**